assets/app/views/builds.html

```diff
@@ -98,8 +114,36 @@
                       </dd>
                     </dl>
                   </span>
-                  <span ng-switch-when="imageChange">
-                    Image change on image repository {{trigger.imageChange.from.name}}:{{trigger.imageChange.tag}}
+                  <span ng-switch-when="imageChange">                    
+                    <div ng-switch="buildConfig.parameters.strategy.type">
+                      <div ng-switch-when="STI">
+                        <dl class="dl-horizontal" ng-if="buildConfig.parameters.strategy.stiStrategy.from&& buildConfig.parameters.strategy.stiStrategy.from.kind!='ImageStreamImage'">
+                          <dt>
+                            On builder update
+                          </dt>
+                          <dd> </dd>
+                          <br>
```

Comment:
yeah it was due to the empty <dd> tag.  so should i remove the br and just trust this will look right once 1756 merges?